bindings/matrix-sdk-ffi/src/calls.rs

```diff
@@ -0,0 +1,39 @@
+use matrix_sdk::RoomCall as SdkRoomCall;
+
+#[derive(uniffi::Object)]
+pub struct RoomCall {
+    pub(crate) inner: SdkRoomCall,
+}
+
+#[uniffi::export]
+impl RoomCall {
+    /// Returns the time when the call was started, i.e. when the first member
```

Comment:
I'm not opposed to renaming everything from `call` to `session`, but there are a couple of things to take into account: I think that if we decide to rename it, then we must rename it everywhere. Having `call` in one place and `session` in others will be rather confusing IMHO. So the question is if this is the right time because doing such renaming implies that we would also need to rename fields in `ruma` (also it means exposing the call type, that by extension should be renamed to the session type).

Is this something you think makes sense to do now?

Reply:
There is definitly some confusion since on a protocol level its called `call`. Especially because the events are called `call.member`
But in ruma there is not that much to rename. I treid to only call things call that are actually calls. (`is_call` ...) but everything that could also be from a different session type is not called using `call`.

But here we are not on the protocol level anymore. The only thing that takes the call.member events is the handler. After that we have our own sturcture which should use `Session`/`Member`/`SessionType` wording. (this also would be the same as in the js sdk)

Except if we have something that only is valid for a room call. (has_active_room_call for instance should use `call`)